app/src/main/java/in/co/rajkumaar/amritarepo/activities/LaunchingActivity.java

```diff
@@ -84,9 +86,12 @@
 public class LaunchingActivity extends BaseActivity
         implements NavigationView.OnNavigationItemSelectedListener {
 
-    private static boolean active = false;
     private boolean doubleBackToExitPressedOnce = false;
     private FirebaseAnalytics mFirebaseAnalytics;
+    private AppUpdateManager appUpdateManager;
+    private Task<AppUpdateInfo> appUpdateInfoTask;
+    private InstallStateUpdatedListener listener;
+    private int request_code = 100;
```

Comment:
By standards, constants are named in uppercase as `REQUEST_CODE = 100`. Can you please change?